src/bindings/js/node/tests/unit/utils.js

```diff
@@ -22,12 +23,36 @@ const testModels = {
 };
 
 module.exports = {
+  compareModels,
   getModelPath,
   downloadTestModel,
   isModelAvailable,
   testModels,
 };
 
+function compareModels(model1, model2, compareNames=true) {
+  let msg = '';
+  if (compareNames && model1.getFriendlyName() !== model2.getFriendlyName()) {
+    msg += 'Friendly names of models are not equal ';
+    msg += `model_one: ${model1.getFriendlyName()}, 
+           model_two: ${model2.getFriendlyName()}`;
+  }
+
+  if (model1.inputs.length !== model2.inputs.length) {
+    msg += 'Number of models\' inputs are not equal ';
+    msg += `model_one: ${model1.inputs.length}, `;
+    msg += `model_two: ${model2.inputs.length}`;
+    throw new Error(msg);
+  }
+
+  if (model1.outputs.length !== model2.outputs.length) {
+    msg += 'Number of models\' outputs are not equal ';
+    msg += `model_one: ${model1.outputs.length}, 
+           model_two: ${model2.outputs.length}`;
+    throw new Error(msg);
+  }
```

Comment:
Didn't catch where `compareNames` using

```suggestion
  const deviations = [];
  
  if (model1.getFriendlyName() !== model2.getFriendlyName())
    deviations.push('Friendly names of models are not equal '
      + `model_one: ${model1.getFriendlyName()}, ` 
      + `model_two: ${model2.getFriendlyName()}`);

  if (model1.inputs.length !== model2.inputs.length)
    deviations.push('Number of models\' inputs are not equal '
      + `model_one: ${model1.inputs.length}, `
      + `model_two: ${model2.inputs.length}`);

  if (model1.outputs.length !== model2.outputs.length)
    deviations.push('Number of models\' outputs are not equal '
      + `model_one: ${model1.outputs.length}, `
      + `model_two: ${model2.outputs.length}`);
  
  if (deviations.length) throw new Error(deviations.join('\n'));
```